src/URDFConverter/UrdfUnity/UrdfUnity/FileManager/FileManager.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.IO;
+using UrdfUnity.Urdf.Models;
+using UrdfUnity.Util;
+using UrdfUnity.Parse;
+using NLog;
+
+
+namespace UrdfUnity.FileManager
+{
+    /// <summary>
+    /// Filetypes we care about in the file manager.
+    /// </summary>
+    public enum FileType
+    {
+        UNKNOWN,
+        URDF,
+        XACRO
+    }
+
+
+    /// <summary>
+    /// This class will be handling the FileIO 
```

Comment:
Nitpick, but let's change the class comment to resemble the others that generally start with a verb: "Handles file input/output for URDF and Xacro files."